[issue#3] add RNG function

Benjibuns opened the pull request:
create new file w/ export and add RNG function fixes #9

Benjibuns committed:
add RNG generator and new lot file

gleekzorp commented on git-play-generator/src/weightedLot.js:
```diff
@@ -0,0 +1,10 @@
+    }
+}
+
+console.log(weightedSlots())
```
Please remove this `console.log`

Benjibuns replied:
Sounds good!

gleekzorp reviewed (changes requested):
Awesome function.  It looks like it will work but let's test it inside the bootstrap instead since your exporting it we really don't want to have a `console.log` inside the `weightedLot.js` file.

Benjibuns commented:
I will fix those and add to boostrap

Benjibuns committed:
remove console.log from weightedLot.js and imported and called file in bootstrap

gleekzorp committed:
Add node_modules to .gitignore and call the function in bootstrap

While I was installing the packages I noticed the gitignore wasn't ignoring the node_modules.

gleekzorp reviewed (changes requested):
Please change your `let` variable to `const`

Benjibuns committed:
pulled updated code and changed let to const

Benjibuns committed:
Merge branch 'issue#3' of https://github.com/Benjibuns/git-play into issue#3

gleekzorp reviewed (approved):
Awesome!

gleekzorp committed:
Merge branch 'main' into issue#3
